src/main/java/com/inqoo/trainingservice/app/models/Course.java

```diff
@@ -5,25 +5,25 @@
 import java.time.Instant;
 
 @Entity
-public class TrainingDetails {
+public class Course {
```

Comment:
Please check if it can private/package private

Reply:
After refactor it will be private/package-private.